### PR TITLE
Add navbar-brand utility information

### PR DESCRIPTION
**Pull request by yash-sr**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/bootstrap/navigation-bar/index.md [hunk 1]
```diff
@@ -52,6 +52,20 @@ You can add buttons on the navbar. The existing Bootstrap Button classes work ho
 <button class="btn navbar-btn">Button</button>
 ```
 
+#### Adding brand logo or name to the navbar
+The .navbar-brand can be applied to most elements, but an anchor works best as some elements might require utility classes or custom styles.
```
Comment (moT01) [inline guide/english/bootstrap/navigation-bar/index.md]:
```suggestion
The `navbar-brand` class can be applied to most elements, but an anchor works best as some elements might require utility classes or custom styles.
```

**Review by moT01 — DISMISSED** [review]
Thanks for the PR @Yashboss - [this is what your PR looks like](https://github.com/freeCodeCamp/freeCodeCamp/blob/b6a0698cfacc101ea783202533b5be6e279f1685/guide/english/bootstrap/navigation-bar/index.md) - You need to put your code snippets in triple backticks. See the code example below yours to see how. It's three backticks with `html` after the third + new line + your code snippet + new line + three more backticks.

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 

@Yashboss Thank you for the PR ✨ 🎉 We're happy to accept these changes and look forward to your future contributions.

Happy coding!

**Pull request merged**